### ISSUE TITLE
Question regarding GC content and Phasomelt parameters

### ISSUE THREAD

**Issue by CuypersBart** [JackAidley/PhasomeIt#8]
Could you maybe recommend how to change the parameters for bacteria with 50% GC content? 
The manual mentions that high-or-low GC content can influence the results, and Campylobacter (for which parameters are recommended) has around 31%, if I am correct.

I have tried on some bacteria with a GC content of 50 and around 96% of all detected SSRs are G-type, of which 79% G7 and 15% G8.

I realise domain knowledge is important for setting these parameters, but it would be nice to at least get an idea for species where domain knowledge is small to non-existing.

**Comment by JackAidley** [comment]
I'm afraid I really have no idea. I created this software for my PhD where I was working only with _Campylobacter_ only, although my supervisor's group went on to use it extensively with _Neisseria_. The settings I chose were based on existing knowledge about variable tracts in _C. jejuni_, which lacks some common error repair mechanisms and thus probably leads to the high variability of these tracts. In other species they may not be so variable, and other mechanisms are used - e.g. much longer tracts, repeats with a longer unit length (e.g. GATC repeats), and - of course - many bacteria use entirely different mechanisms. The best thing, I think, would be to look at the literature on variability mechanisms in the bacteria you are interested in and look for tracts of similar length/type.

But as a baseline, I would consider setting the values to detect SSRs which are expected to occur in low numbers by chance, and then for species with a high/low GC contact using the filter setting with W or S to filter out either GC or AT tracts (respectively) selectively to a higher length. For example, with _C. jejuni_ without filtering most of the results are shorter AT tracts and these tracts do seem to be significantly less variable than GC tracts of the same length so are probably not biologically important anyway.
